datastore/datastore.go

```diff
@@ -134,7 +135,8 @@ func makeDefaultScopes() map[string]*ScopeCfg {
 			Provider: string(store.BOLTDB),
 			Address:  defaultPrefix + "/local-kv.db",
 			Config: &store.Config{
-				Bucket: "libnetwork",
+				Bucket:            "libnetwork",
+				ConnectionTimeout: time.Minute,
```

Comment:
There is no one good number for timeouts. But in the earlier PR that @coolljt0725 pushed the timeout was set to 2 minutes. I am ok with 1 minute though if it was sufficient in the stress tests you did.